docs/docsite/rst/community/documentation_contributions.rst

```diff
@@ -0,0 +1,166 @@
+.. _community_documentation_contributions:
+
+*****************************************
+Contributing to the Ansible Documentation
+*****************************************
+
+Ansible has a lot of documentation and a small team of writers. Community support helps us keep up with new features, fixes, and changes.
+
+Improving the documentation is an easy way to make your first contribution to the Ansible project. You don't have to be a programmer, since our documentation is written in YAML (module documentation) or reStructured Text. If you're using Ansible, you know YAML - playbooks are written in YAML. And rST is mostly just text. You don't even need git experience, if you use the ``Edit on GitHub`` option.
```

Comment:
Shouldn't it be `YAML-playbooks` or `YAML playbooks`, and not `YAML - playbooks`?

Reply:
`or reStructured Text`
`or reStructured Text (RST)`